static/app/components/events/interfaces/crashContent/exception/sourcemapsWizard.tsx

```diff
@@ -0,0 +1,116 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import EmptyMessage from 'sentry/components/emptyMessage';
+import ExternalLink from 'sentry/components/links/externalLink';
+import Panel from 'sentry/components/panels/panel';
+import {IconClose, IconFlag} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import localStorage from 'sentry/utils/localStorage';
+import useOrganization from 'sentry/utils/useOrganization';
+
+const LOCAL_STORAGE_KEY = 'issues-sourcemap-wizard-hide-until';
+const DISMISS_TIME = 1000 * 60 * 60 * 24 * 7; // 1 week
+
+const wizardCommand = 'npx @sentry/wizard@latest -i sourcemaps';
+
+function getHideUntilTime() {
+  return Number(localStorage.getItem(LOCAL_STORAGE_KEY)) || 0;
+}
+
+function setHideUntilTime(offset: number) {
+  localStorage.setItem(LOCAL_STORAGE_KEY, String(Date.now() + offset));
+}
+
+function clearHideUntilTime() {
+  localStorage.removeItem(LOCAL_STORAGE_KEY);
+}
+
+export default function SourceMapsWizard() {
+  const organization = useOrganization();
+  const [isHidden, setIsHidden] = useState(() => {
+    const hideUntilTime = getHideUntilTime();
+    if (hideUntilTime && Date.now() < hideUntilTime) {
+      return true;
+    }
+    clearHideUntilTime();
+    return false;
+  });
+
+  if (isHidden) {
+    return null;
+  }
+
+  return (
+    <Panel dashedBorder data-test-id="sourcemaps-wizard">
+      <CloseButton
```

Comment:
nit: this button isn't perfectly square, you'll want to add custom styling to make it so: 


![CleanShot 2023-07-24 at 13 39 54](https://github.com/getsentry/sentry/assets/10888943/890fcc02-9fcc-4c26-b092-04e8adfaa56a)